article/using-modern-agricultural-technology-to-increase-production-food-security-and-profitability/index.html

```diff
@@ -0,0 +1,496 @@
+
```

Comment:
What is going on with this file?

Reply:
Yikes! Must have forgotten to drop the redirect code in there...

Reply:
This is an archived article that doesn't show up in the article list unless you check the "Include Archived Results" filter. We're not sure why it was archived if it gets high traffic, but I recreated it by hand so I could add the redirect.

Reply:
☝️ since it doesn't show up in the default stories listing, the static generator doesn't "see" it.